### ISSUE TITLE
VariationActivationError upon selecting variation of theme to assign

### ISSUE THREAD

**Issue by Ash-Matter** [bigcommerce/stencil-cli#547]
### Expected behavior
Provide success message of activation of the theme and its variation

### Actual behavior
Throwing 502 error as follows:
```not ok -- VariationActivationError: Response Error: 502 Bad Gateway see details below:
unknown error```

### Steps to reproduce behavior
Run `stencil push`. We manage a number of BC projects and it seems to only be happening to this project. If someone would like to get in touch for more detailed information, I can provide outside of this forum.

**Comment by Ash-Matter** [comment]
Theme only contains 1 variation - may or may not be useful information

**Comment by carsonreinke** [comment]
@Ash-Matter so is the issue you are not receiving success of activation of a variation or an issue activating a variation?

**Comment by Ash-Matter** [comment]
@carsonreinke seems to be just the success - the function itself appears to still be activating the theme and the variation of that theme in BC. Its just the feedback of that - instead of the nice green "complete", its erroring.

**Comment by carsonreinke** [comment]
@Ash-Matter can you see the theme uploaded on the Storefront?

**Comment by Ash-Matter** [comment]
@carsonreinke yep, everything functionality is fine, just the feedback from that is broken as far as I'm aware.
